test/players.js

```diff
@@ -11,6 +11,43 @@ describe('GET /players', () => {
             .get('/players')
             .expect(httpStatus.OK);
     });
+
+    it('should return all players sorted by id', async () => {
+        const res = await supertest(app)
+            .get('/players')
+            .query({
+                sortField: 'id',
+                sortDirection: 'ASC'
+            })
+            .expect(httpStatus.OK);
+
+        const sortedPlayers = await models.player.findAll({
+            order: [
+                ['id', 'ASC']
+            ],
+            raw: true
+        });
+
+        const playersIdSortedByParams = res.body.players.map((player) => {
+            return { id: player.id }
+        });
+
+        const playersIdSortedManually = sortedPlayers.map((player) => {
+            return { id: player.id }
+        });
+
+        assert.deepStrictEqual(playersIdSortedByParams, playersIdSortedManually, 'Should sort players by ascending id');
+    });
+
+    it('should return validation error', async () => {
```

Comment:
Not enough specific in the case name, you already have one test for validation error. Either be more specific in the case name or extract these two cases to a separate describe for sort scenario